manager/orchestrator/taskreaper/task_reaper.go

```diff
@@ -96,10 +96,10 @@ func (tr *TaskReaper) Run(ctx context.Context) {
 			// Serviceless tasks can be cleaned up right away since they are not attached to a service.
 			tr.cleanup = append(tr.cleanup, t.ID)
 		}
-		// tasks with desired state REMOVE that have progressed beyond SHUTDOWN can be cleaned up
+		// tasks with desired state REMOVE that have progressed beyond COMPLETE can be cleaned up
 		// right away
 		for _, t := range removeTasks {
-			if t.Status.State >= api.TaskStateShutdown {
+			if t.Status.State >= api.TaskStateCompleted {
```

Comment:
nit: Why is this not `> api.TaskStateRunning`?

That's pretty much the norm throughout the codebase to see whether a Task is running or not:
```
$ ack '> api.TaskStateRunning'
cmd/swarmctl/task/print.go:48:		if !all && t.DesiredState > api.TaskStateRunning {
cmd/swarmctl/task/inspect.go:40:	if t.Status.State > api.TaskStateRunning {
manager/dispatcher/dispatcher_test.go:589:		return s > api.TaskStateRunning
manager/dispatcher/assignments.go:229:			if t.Status.State > api.TaskStateRunning {
manager/scheduler/nodeinfo.go:111:		if t.DesiredState <= api.TaskStateRunning && oldTask.DesiredState > api.TaskStateRunning {
manager/scheduler/nodeinfo.go:116:		} else if t.DesiredState > api.TaskStateRunning && oldTask.DesiredState <= api.TaskStateRunning {
manager/scheduler/scheduler.go:73:		if t.Status.State < api.TaskStatePending || t.Status.State > api.TaskStateRunning {
manager/scheduler/scheduler.go:208:	if t.Status.State < api.TaskStatePending || t.Status.State > api.TaskStateRunning {
manager/scheduler/scheduler.go:245:	if t.Status.State > api.TaskStateRunning {
manager/allocator/network.go:627:		if t.Status.State > api.TaskStateRunning {
manager/allocator/network.go:794:	if t.Status.State > api.TaskStateRunning || isDelete {
manager/orchestrator/replicated/tasks.go:54:						if t.DesiredState > api.TaskStateRunning {
manager/orchestrator/replicated/tasks.go:95:			if t.DesiredState > api.TaskStateRunning {
manager/orchestrator/replicated/tasks.go:121:	if t.DesiredState > api.TaskStateRunning {
manager/orchestrator/replicated/tasks.go:142:	if t.Status.State > api.TaskStateRunning ||
manager/orchestrator/replicated/tasks.go:153:	if t.DesiredState > api.TaskStateRunning {
manager/orchestrator/replicated/tasks.go:175:	if t.Status.State > api.TaskStateRunning ||
manager/orchestrator/replicated/update_test.go:78:				} else if task.DesiredState > api.TaskStateRunning {
manager/orchestrator/constraintenforcer/constraint_enforcer.go:104:		if t.DesiredState < api.TaskStateAssigned || t.DesiredState > api.TaskStateRunning {
manager/orchestrator/constraintenforcer/constraint_enforcer.go:150:					if t == nil || t.DesiredState > api.TaskStateRunning {
manager/orchestrator/taskinit/init.go:62:			if t.DesiredState != api.TaskStateReady || t.Status.State > api.TaskStateRunning {
manager/orchestrator/update/updater.go:496:		if original.DesiredState > api.TaskStateRunning {
manager/orchestrator/update/updater.go:504:			if t.DesiredState > api.TaskStateRunning {
manager/orchestrator/update/updater_test.go:386:			} else if task.DesiredState > api.TaskStateRunning {
manager/orchestrator/restart/restart.go:86:		if t.DesiredState > api.TaskStateRunning {
manager/orchestrator/restart/restart.go:124:	if t.DesiredState > api.TaskStateRunning {
manager/orchestrator/restart/restart.go:173:	if (n != nil && n.Status.State == api.NodeStatus_DOWN) || t.Status.State > api.TaskStateRunning {
manager/orchestrator/global/global.go:181:	if t.DesiredState > api.TaskStateRunning {
manager/orchestrator/global/global.go:196:	if t.Status.State > api.TaskStateRunning {
manager/orchestrator/global/global.go:208:	if t.DesiredState > api.TaskStateRunning {
manager/orchestrator/global/global.go:213:	if t.Status.State > api.TaskStateRunning {
manager/orchestrator/global/global.go:496:				if t == nil || t.DesiredState > api.TaskStateRunning {
```